/**:
  ros__parameters:
    left_wheel_signal:
      type: axis
      index: 1                    # Left Stick
    right_wheel_signal:
      type: axis
      index: 3                    # Right Stick
    raise_front_arm_signal:
      type: button
      index: 5                    # R
    lower_front_arm_signal:
      type: button
      index: 7                    # ZR
    raise_back_arm_signal:
      type: button
      index: 4                    # L
    lower_back_arm_signal:
      type: button
      index: 6                    # ZL
    dig_front_drum_signal:
      type: button
      index: 2                    # A
    dump_front_drum_signal:
      type: button
      index: 3                    # X
    dig_back_drum_signal:
      type: button
      index: 1                    # B
    dump_back_drum_signal:
      type: button
      index: 0                    # Y
    stop_routine_signal:
      type: button
      index: 8                    # -
    auto_drive_routine_signal:
      type: axis
      index: 4                    # D-Pad Left
      accept: 1.0
    auto_dig_routine_signal:
      type: axis
      index: 5                    # D-Pad Up
      accept: 1.0
    auto_dump_routine_signal:
      type: axis
      index: 4                    # D-Pad Right
      accept: -1.0
    auto_dock_routine_signal:
      type: axis
      index: 5                    # D-Pad Down
      accept: -1.0
    full_autonomy_routine_signal:
      type: button
      index: 9                    # +
